packages/untp-test-suite/src/interfaces/utils/logger.ts

```diff
@@ -0,0 +1,127 @@
+import chalk from 'chalk';
+import Table from 'cli-table3';
+import { getPackageVersion } from '../../utils/common.js';
+
+export enum TestSuiteResultEnum {
+  PASS = 'PASS',
+  FAIL = 'FAIL',
+  WARN = 'WARN',
+}
+
+export function getLogStatus(testSuiteResults: any[]) {
```

Comment:
`testSuiteResults` should be not `any[]`

Reply:
Yes, I agree that it should have a specific type. The type should be defined in the `core` service because there is a place to create the data and pass it to the `testSuiteResults` parameter here, and that type should go with PR [#31](https://github.com/uncefact/tests-untp/pull/31). If I define it here, it might duplicate the type for the `testSuiteResults` parameter.

I have reviewed the PR #31 created by Yen. In it, she has defined a type that I think can be suitable for the `testSuiteResults` parameter here.

```js
// untp-test-suite/src/core/types/index.ts

export interface ICredentialTestResult {
  credentialType: string;
  version: string;
  path: string;
  result: string;
  warnings?: string[];
  error?: string[];
}
```

If we have a task to integrate all things together and optimize the code, I can now create an interface for the `testSuiteResults` parameter. Let's not worry about duplicate types for now.